Apps/AdminWebClient/src/ClientApp/src/components/core/CommunicationTable.vue

```diff
@@ -232,8 +233,15 @@ export default class CommunicationTable extends Vue {
     private edit(item: Communication) {
         console.log(item);
         this.isNewCommunication = false;
-        this.editedBanner = item;
-        this.editedEmail = item;
+        if (item.communicationTypeCode === CommunicationType.Email) {
```

Comment:
ah, there you go. I knew it would cause problems at some point =)

Reply:
you called it!